lib/FCM/System/Make/Extract.pm

```diff
@@ -1080,7 +1080,6 @@ sub _target_update {
     ) {
         return _target_delete($attrib_ref, $m_ctx, $ctx, $target);
     }
-    $target->set_status($target->ST_UNCHANGED);
     my $path = _target_update_source($attrib_ref, $m_ctx, $ctx, $target);
```

Comment:
FYI: the new test passes when just the `target->set_status(...)` and `my $path = _target_update_source(...)` lines are swapped.